config/forms/2021_2022.json

```diff
@@ -196,7 +196,7 @@
                 }
               }
             },
-            "age1": {
+            "person_1_age": {
```

Comment:
I'm not sure how this conflicts with @kosiakkatrina 's data storage ticket

Reply:
It shouldn't do, this is changing the `page` name which is what the validations depend on, whereas @kosiakkatrina is changing `questions` which is what the data storage depends on.